### ISSUE TITLE
I'm trying use

### ISSUE THREAD

**Issue by bigbruno** [caux/gekkostuff#1]
I'm trying use wvf indicator, but i think not complete, just incomplete or is my fault?

/home/bruno/gekko/gekko-develop/strategies/indicators/WVF.js:9
    this.pd = settings.pd;      // title="LookBack Period Standard Deviation High"
                       ^

TypeError: Cannot read property 'pd' of undefined

**Comment by caux** [comment]
Hello, you need the toml as well, which I didn't commit

Just add this to the strat parameters (and tweak the variables):
```
pd = 22
bbl = 20
mult = 2
lb = 50
ph = 0.85 
```

**Comment by bigbruno** [comment]
Thank you, now the problem is gone, but have another, now isBottom always show false.

Seeing the code, not need implement Lowest Percentile?

**Comment by caux** [comment]
No, Lowest Percentile wasn't used in the original code. It may sound obvious, but did you add the indicator in the strat.init? I'm gonna commit an example use, plus the toml soon to make it easier for people to figure it out.

To tweak the vars I recommend you go on TradingView, add the original script and change the variables there until you get what you want, then just copy them over to your strat parameters.

**Comment by bigbruno** [comment]
Hello you maintain working with gekko strats?

I'm improved mounirs-2, but need more improvements, maybe you wanna talk about this, my telegram is @bigbruno
